Sources/AsyncHTTPClient/SSLContextCache.swift

```diff
@@ -12,19 +12,19 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Dispatch
 import Logging
 import NIO
 import NIOConcurrencyHelpers
 import NIOSSL
 
 class SSLContextCache {
-    private var state = State.activeNoThread
+    private var state = State.active
     private let lock = Lock()
     private var sslContextCache = LRUCache<BestEffortHashableTLSConfiguration, NIOSSLContext>()
-    private let threadPool = NIOThreadPool(numberOfThreads: 1)
+    private let threadPool = DispatchQueue(label: "io.github.swift-server.AsyncHTTPClient.SSLContextCache")
```

Comment:
I'm not sure if I approve staying with the name `threadPool`. In this context threadPool implies a NIOThreadPool to me and I get high blood pressure if I don't immediately find a `threadPool.shutdown` somewhere in the file.

Reply:
Suggestion `offloadQueue`?

Reply:
done, thanks